demo/chicago-bikeshare-dashboard/run.py

```diff
@@ -27,12 +27,18 @@ def get_count_ride_type():
     """,
         con=connection_string,
     )
-    fig_m, ax = plt.subplots()
-    ax.bar(x=df["rideable_type"], height=df["n"])
-    ax.set_title("Number of rides by bycycle type")
-    ax.set_ylabel("Number of Rides")
-    ax.set_xlabel("Bicycle Type")
-    return fig_m
+    return gr.BarPlot.update(
+        value=df,
+        x="rideable_type",
+        y='n',
+        title="Number of rides per bicycle type",
+        y_title="Number of Rides",
+        x_title="Bicycle Type",
+        vertical=False,
+        tooltip=['rideable_type', "n"],
+        height=300,
+        width=300,
+    )
```

Comment:
Wouldn't it be better to have these in the definition of `gr.Barplot` below and only `update()` the `value` parameter?